pkgs/tools/filesystems/go-mtpfs/default.nix

```
@@ -29,5 +29,6 @@ buildGoModule rec {
     homepage = "https://github.com/hanwen/go-mtpfs";
     license = licenses.bsd3;
     maintainers = with maintainers; [ aaronjheng ];
+    broken = (stdenv.isDarwin && setenv.isx86_64);
```

also broken on aarch64-darwin https://hydra.nixos.org/build/198390352
```suggestion
    broken = stdenv.isDarwin;
```